examples/ts-solid-chat/src/routes/__root.tsx

```
@@ -1,6 +1,6 @@
 import { HeadContent, Scripts, createRootRoute } from '@tanstack/solid-router'
-import { TanStackRouterDevtoolsPanel } from '@tanstack/solid-router-devtools'
-import { TanStackDevtools } from '@tanstack/solid-devtools'
+// Devtools are dynamically imported on the client to avoid server-side bundling
+import { onMount, createSignal } from 'solid-js'
```

ESLint

[error] Member 'createSignal' of the import declaration should be sorted alphabetically.

(sort-imports)